ntoskrnl/mm/ARM3/sysldr.c

```diff
@@ -2814,6 +2820,80 @@ MmCheckSystemImage(IN HANDLE ImageHandle,
     return Status;
 }
 
+
+PVOID
+NTAPI
+LdrpFetchAddressOfSecurityCookie(PVOID BaseAddress, ULONG SizeOfImage)
+{
+    PIMAGE_LOAD_CONFIG_DIRECTORY ConfigDir;
+    ULONG DirSize;
+    PVOID Cookie = NULL;
+
+    /* Check NT header first */
+    if (!RtlImageNtHeader(BaseAddress)) return NULL;
```

Comment:
```suggestion
    if (!RtlImageNtHeader(BaseAddress))
        return NULL;
```

Reply:
No, following the style of the surrounding code.